### DISCUSSION TITLE
helloi am new here creating apps for amazfit band 7.

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#62] [Q&A]
i wanted to get codes of water drinking reminders app, and bmi app codes please please . i wanted to learn from those apps please give me . i want 1.0 version of those apps

**Comment by juan518munoz** [comment]
There are three example apps to use for guidance, fetch API, Calories and notes app.

The ones you mention are not written using the zeppOS framework.

**Comment by MyoungXUE** [comment]
Hello, we currently provide 3 Mini Programs that cover some scenarios and can be used as a reference

https://docs.zepp.com/docs/1.0/samples/
